Makefile

```diff
@@ -263,6 +263,7 @@ CGOFLAG_TSH ?= $(CGOFLAG)
 ELECTRON_BUILDER_ARCH_amd64 = x64
 ELECTRON_BUILDER_ARCH = $(or $(ELECTRON_BUILDER_ARCH_$(ARCH)),$(ARCH))
 
+BUILDFLAGS += $(if $(filter $(TELEPORT_BUILD_TYPE),community),-ldflags="-X github.com/gravitational/teleport/lib/modules.teleportBuildType=community")
```

Comment:
I think we need to get a little more sophisticated here.

It looks like if you specify `-ldflags` more than once, the last one wins.

There are other parts of the Makefile where we do specify `-ldflags`, meaning we'd lose some settings.

We probably need an `LDFLAGS` variable that we use everywhere, and just append to it (`+=`) in this case.

Reply:
I updated. I believe i got every instance of `-ldflags` and appending correctly. Test _my_ changes and it works as expected with/without but would love a second pair of eyes